mvc/controllers/actions.rst

```diff
@@ -1,8 +1,77 @@
-.. include:: /../common/stub-topic.txt
+Actions and Action Results
```

Comment:
Since this article covers the basics of controllers and actions I think the title should be "Controllers, Actions and Action Results".